sunspot/spec/integration/field_lists_spec.rb

```diff
@@ -31,6 +31,22 @@
     end
   end
 
+  it 'when listing existing text fields it should not raise Sunspot::UnrecognizedFieldError' do
```

Comment:
can we change title wording to "does not raise ...." ?

Reply:
@serggl  Yes, definetely will do.

Reply:
@serggl  done.